src/clientmetricreport/DefaultClientMetricReport.ts

```diff
@@ -238,6 +238,11 @@ export default class DefaultClientMetricReport implements ClientMetricReport {
     bytesSent: { transform: this.bitsPerSecond, type: SdkMetric.Type.VIDEO_SENT_BITRATE },
     droppedFrames: { transform: this.countPerSecond, type: SdkMetric.Type.VIDEO_DROPPED_FPS },
     qpSum: { transform: this.countPerSecond, type: SdkMetric.Type.VIDEO_SENT_QP_SUM },
+    googFrameHeightSent: {
```

Comment:
This works for  chrome legacy getStats, could we have the same thing for standard stats ? ( aka, have we tested the code in safari or Firefox ?)

Reply:
Firefox and Safari does not support video resolution as far as i know

Reply:
I tested in FF and Safari, we can get frameHeight and frameWidth in the inbound-rtp and outbound-rtp report. Will add these into the metrics here.